stubs/tensorflow/tensorflow/summary.pyi

```diff
@@ -0,0 +1,61 @@
+import abc
+from _typeshed import Incomplete
+from collections.abc import Callable, Iterator
+from contextlib import AbstractContextManager, contextmanager
+from typing import Any, Literal
+from typing_extensions import Self
+
+import numpy as np
+import numpy.typing as npt
+import tensorflow as tf
+from tensorflow.experimental.dtensor import Mesh
+
+class SummaryWriter(metaclass=abc.ABCMeta):
+    def as_default(self, step: int | None = None) -> AbstractContextManager[Self]: ...
+    def close(self) -> None: ...
+    def flush(self) -> None: ...
+    def init(self) -> None: ...
+    def set_as_default(self, step: int | None = None) -> None: ...
+
+def audio(
+    name: str,
+    data: tf.Tensor,
+    sample_rate: int | tf.Tensor,
+    step: int | tf.Tensor | None = None,
+    max_outputs: int | tf.Tensor | None = 3,
+    encoding: Literal["wav"] | None = None,
+    description: str | None = None,
+) -> bool: ...
+def create_file_writer(
+    logdir: str,
+    max_queue: int | None = None,
+    flush_millis: int | None = None,
+    filename_suffix: str | None = None,
+    name: str | None = None,
+    experimental_trackable: bool = False,
+    experimental_mesh: Mesh | None = None,
+) -> SummaryWriter: ...
+def create_noop_writer() -> SummaryWriter: ...
+def flush(writer: SummaryWriter | None = None, name: str | None = None) -> tf.Operation: ...
+def graph(graph_data: tf.Graph | tf.compat.v1.GraphDef) -> bool: ...
+def histogram(
+    name: str, data: tf.Tensor, step: int | None = None, buckets: int | None = None, description: str | None = None
+) -> bool: ...
+def image(
+    name: str,
+    data: tf.Tensor | npt.NDArray[np.uint8 | np.number[Any]],
```

Comment:
Is there any difference in `np.uint8 | np.number[Any]` vs  `np.number[Any]`?

Reply:
I'd also lean to have any npt.NDArray types with a specific data type to be defined in _aliases so we can have IntNDArray, FloatNDArray, AnyNDArray, etc and be consistent on our definitions.

Reply:
I've modified this in a2e5fcaac, I'm not sure if that's what you meant.